tests/test_pdo.py

```diff
@@ -270,3 +271,31 @@ def test_start_pdos_number_of_network_exception(mocker, motion_controller):
         mc.capture.pdo.start_pdos()
     with pytest.raises(IMException):
         mc.capture.pdo.start_pdos(COMMUNICATION_TYPE.Ethercat)
+
+
+def skip_if_pdo_padding_is_not_available(mc, alias):
+    pdo_poller_fw_version = "2.5.0"
```

Comment:
Maybe you should modify config.json to install FW 2.5.0.

Reply:
Done